optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/java/org/optaplanner/workbench/screens/domaineditor/client/handlers/planner/PlannerDomainHandler.java

```diff
@@ -43,7 +47,7 @@ public int getPriority() {
 
     @Override
     public DomainEditor getDomainEditor( boolean newInstance ) {
-        PlannerDomainEditor domainEditor = IOC.getBeanManager().lookupBean( PlannerDomainEditor.class ).newInstance();
+        PlannerDomainEditor domainEditor = plannerDomainEditorProvider.get();
         domainEditor.setHandler( this );
```

Comment:
@csadilek Hmm good point. Looking at the source code (optaplanner-wb, drools-wb, guvnor, kie-wb-common) it looks like this method is no longer in use and should therefore be removed (if I'm not missing anything obvious). Wdyt?